website/docs/markdown-features.mdx

```diff
@@ -1043,4 +1043,19 @@ or
 [![](./assets/docusaurus-asset-example-banner.png)](./assets/docusaurus-asset-example-pdf.pdf)
 ```
 
-[![](./assets/docusaurus-asset-example-banner.png)](./assets/docusaurus-asset-example-pdf.pdf)
\ No newline at end of file
+[![](./assets/docusaurus-asset-example-banner.png)](./assets/docusaurus-asset-example-pdf.pdf)
+
+### Inline SVGs
+
+Docusaurus support inlining SVGs out of the box.
+
+```js
+
+import SVG from "../static/img/docusaurus.svg"
+
+<SVG />
```

Comment:
```suggestion
import DocusaurusSvg from "./docusaurus.svg"

<DocusaurusSvg />
```

Better comp name + relative import in same folder. Relative import from static folder is risky because once the doc becomes versioned that relative path may change and break.